vspec2x.py

```diff
@@ -178,6 +179,7 @@ def main(arguments):
                                         data_type_tree=data_type_tree)
             vspec.merge_tree(tree, othertree)
 
+        vspec.check_type_usage(tree, VSSTreeType.SIGNAL_TREE, data_type_tree)
```

Comment:
To support multiple type files I had to postpone type check until all trees have been merged, so that we only check on final tree.